ansible/roles/mixing/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+# Mixing
+
+- name: Ensure jq is installed
+  ansible.builtin.apt:
+    name: jq
+    state: present
+    update_cache: true
+  become: true
+
+- name: Create CoinJoin wallets
+  ansible.builtin.command:
+    cmd: dash-cli createwallet {{ coinjoin_wallet_name }} false false "" false true
+    creates: "{{ dashd_home }}/.dashcore/{{ dash_network_name if dash_network == 'devnet' else 'testnet3' }}/wallets/{{ coinjoin_wallet_name }}/wallet.dat"
```

Comment:
If a mixer node restarts, this step will (correctly) be skipped because the wallet file already exists. Will the wallets still be loaded after the restart?

Reply:
Yes due to the "true" at the end, it's "load_on_startup"

https://docs.dash.org/projects/core/en/stable/docs/api/remote-procedure-calls-wallet.html#createwallet